### PR TITLE
use shorter VM name for PacketFence ZEN

### LINKED ISSUE #5146: Add domain on PacketFence-Zen
Default PF zen hostname: PacketFence-ZEN

If we keep the %h for the domain configuration it goes over 14 chars.

### PR DESCRIPTION
**Pull request by julsemaan**
# Description
Make the PacketFence ZEN VM name shorter (now `PF-ZEN`)

# Impacts
new ZEN VMs

# Issue
fixes #5146 

# Delete branch after merge
YES

### CODE REVIEW

File: addons/ZEN-vagrant-builder/installer/install-pf.sh [hunk 1]
```diff
@@ -19,9 +19,9 @@ yum install perl -y
 yum install --enablerepo=$PFREPO $PFPACKAGE -y
 
 # Setting the hostname
-hostname PacketFence-ZEN
-echo "PacketFence-ZEN" > /etc/hostname
-echo "NETWORKING=yes HOSTNAME=PacketFence-ZEN" > /etc/sysconfig/network
+hostname PF-ZEN
```
Comment (extrafu) [inline addons/ZEN-vagrant-builder/installer/install-pf.sh]: Shouldn't we just do that and leave the rest as-is?

Comment (julsemaan) [reply addons/ZEN-vagrant-builder/installer/install-pf.sh]: To make things cleaner and more consistent I changed everything but if you prefer the full name to be used elsewhere (like the VM ESX name), I can revert those

Comment (extrafu) [reply addons/ZEN-vagrant-builder/installer/install-pf.sh]: That's my preference, but it doesn't mean we should do it!

I think "PF ZEN" isn't nice-looking for ESX name and others. For the hostname, heck I don't mind - we could even call it bob.

Comment (julsemaan) [reply addons/ZEN-vagrant-builder/installer/install-pf.sh]: Thinking about it again, `packetfence` is 12 characters which meets the requirement for the fix.

So I'll just change the hostname to that and leave the rest as is

**Comment by nqb** [comment]
Not related to this PR but you should use:
```
yum localinstall http://packetfence.org/downloads/PacketFence/RHEL7/packetfence-release-7.stable.noarch.rpm
```
in `addons/ZEN-vagrant-builder/installer/install-pf.sh`

**Comment by julsemaan** [comment]
I adjusted the VM name to `packetfence` and left all the rest to PacketFence-ZEN to have it cleaner in the ESX and download links.

I didn't do the release RPM install change, its unrelated to this PR

**Pull request merged**